analyses/snv-callers/scripts/01-setup_db.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python
+#!/bin/env python3
```

Comment:
I do not think this change is necessary, as python3 is the default install in docker.
```suggestion
#!/bin/env python
```

Reply:
@jharenza, I have updated the run script,  now  named `run_caller_consensus_analysis.sh` and  the TBM  script `03-calculate-tmb.R` to accept a parsable option for the project acronym prefix that is prepended to all input and output files names hardcoded in the scripts. 
The default python the the docker container is python v2.7.13. The reason why script `01-setup_db.py` is runs successfully is because it is called inside run bash script with python3, not python. I have reverted the shebang back to python.